zboxcore/sdk/uploadworker_test.go

```diff
@@ -0,0 +1,65 @@
+package sdk
```

Comment:
I'm not sure if unit tests exist elsewhere (i couldn't find them) if they do let me know and i'll move these out

Reply:
07f3f8837df504d2f5f164992b34f4542f60810e